### DISCUSSION TITLE
where is Publish as Tool?

### DISCUSSION THREAD

**Discussion by huu3301** [langgenius/dify#5402] [Help] [answered]
I can't find where the Publish as Tool is.
And the version is 0.6.11.

![img_v3_02c0_09a15b7d-4073-4cb5-a1c4-70393ab0beag](https://github.com/langgenius/dify/assets/44219645/676a0f00-83df-4485-80a8-08bbc3022e01)

**Comment by perzeuss** [comment, marked as answer]
You can only do that with workflows, I assume you are building a chatflow?

**Comment by huu3301** [comment]
@perzeuss 
The problem has been resolved. 
By the way, will it support publishing a agent as a tool in the future?
Thank you.

**Reply by perzeuss** [reply]
I don't know, you can submit a feature request. If the team is working on it, they'll add a note on the feature request.
